source/features/latest-tag-button.tsx

```diff
@@ -37,26 +39,59 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 	return tags[0];
 }, {
 	expiration: 1,
-	cacheKey: () => __featureName__ + ':' + getRepoURL()
+	cacheKey: () => __featureName__ + '_tags:' + getRepoURL()
 });
 
-function getTagLink(latestRelease: string): HTMLAnchorElement {
+async function currentBranchIsBleedingEdge(latestTag: string, defaultBranch: string, currentBranch: string): Promise<boolean> {
+	// Only default branch can be cutting edge (i.e., ahead of latest release)
+	if (currentBranch !== defaultBranch) {
+		return false;
+	}
+
+	// Find the last commits from the default branch and latest tag
+	const repository = await api.v4(`
```

Comment:
```suggestion
	const {repository} = await api.v4(`
```

This lets your skip the unsightly `repository.repository...` later

Reply:
Cheers, didn't know that